test/contrast_kwargs.jl

```diff
@@ -0,0 +1,20 @@
+include(raw"../src/ImageView.jl")
```

Comment:
This file doesn't get tested.

Instead of `include(raw"../src/ImageView.jl")` here. You should add `include("contrast_kwargs.jl")` in `test/runtests.jl`.

Or, just add these tests in `test/contrast.jl` would be fine.

Reply:
Thanks, I have never done tests in Julia before.